script.ps1

```diff
@@ -0,0 +1,23 @@
+$mergeIds = (git log --pretty=format:'%h' --merges)
```

Comment:
Was this checked in by accident?

Reply:
Oh, yes this was checked in by accident. I committed the source code using vscode and it somehow committed not just the staged files.